x/dao/keeper/msg_server_report_pop_result.go

```diff
@@ -19,6 +19,12 @@ func (k msgServer) ReportPopResult(goCtx context.Context, msg *types.MsgReportPo
 		return nil, errorsmod.Wrapf(types.ErrInvalidChallenge, err.Error())
 	}
 
+	if msg.Challenge.GetSuccess() {
```

Comment:
shouldn't that be the other way around?
```
        if msg.Challenge.GetSuccess() {
                util.GetAppLogger().Info(ctx, "PoP at height %v was successful", msg.Challenge.GetHeight())
        } else {
                util.GetAppLogger().Info(ctx, "PoP at height %v was unsuccessful", msg.Challenge.GetHeight())
        }
```

Reply:
If msg.Challenge.GetSuccess() == True, then output:  successful
otherwise: unsuccessful. 
I think it's correct

Reply:
yea that is my suggestion, but in your code it says `If msg.Challenge.GetSuccess() == True, then "PoP at height %v was unsuccessful"